powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthRestClient.java

```diff
@@ -286,6 +286,7 @@ public CreateActivationResponse createActivation(String userId, Date timestampAc
         request.setEncryptedData(encryptedData);
         request.setMac(mac);
         request.setNonce(nonce);
+        request.setGenerateRecoveryCodes(true);
```

Comment:
I think we should leave this to default (`false`).

Reply:
Set default `CreateActivationRequest#generateRecoveryCodes` to `true`.